docs/spec/mint/begin_block.md

```diff
@@ -1,20 +1,20 @@
 # Begin-Block
 
-Inflation occurs at the beginning of each block, however minting parameters
-are only calculated once per hour.
+Minting parameters are recalculated and inflation
+paid at the beginning of each block.
 
 ## NextInflationRate
 
-The target annual inflation rate is recalculated at the first block of each new
-hour. The inflation is also subject to a rate change (positive or negative)
+The target annual inflation rate is recalculated each block.
```

Comment:
Does this warrant any changes to the distribution spec (e.g. `Affects on Staking`)?